### ISSUE TITLE
Improve accessibility  - Replace "Up" and "Down"

### ISSUE THREAD

**Issue by adjn** [BagToad/Zendesk-Link-Collector#16]
Currently the "Reorder" buttons say "Up" and "Down". For improved accessibility to non-English speakers, those should be converted to up and down arrows.
